src/clientdlg.cpp

```diff
@@ -1001,6 +1000,7 @@ void CClientDlg::ShowConnectionSetupDialog()
     // show connect dialog
     bConnectDlgWasShown = true;
     ConnectDlg.show();
+    ConnectDlg.setWindowTitle ( MakeClientNameTitle( tr ( "Connect" ) , pClient->strClientName ) );
```

Comment:
Sorry, spaces after the method name and before the `(`, too, please, in `MakeClientNameTitle(` - throughout.

Reply:
Fine with me, will do, I was just orienting on the style in the clientdlg constructor.